recipes-bsp/u-boot/u-boot-imx-common.inc

```diff
@@ -3,9 +3,9 @@ DESCRIPTION = "i.MX U-Boot suppporting i.MX reference boards."
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://Licenses/gpl-2.0.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
-SRCBRANCH = "lf-5.4.y_v2019.04"
+SRCBRANCH = "imx_v2020.04_5.4.24_2.1.0"
 SRC_URI = "git://source.codeaurora.org/external/imx/uboot-imx.git;protocol=https;branch=${SRCBRANCH}"
-SRCREV = "228843cdf5435d4bd69f42a6015f78761ff4cc0d"
+SRCREV = "4979a99482f7e04a3c1f4fb55e3182395ee8f710"
```

Comment:
The `u-boot-imx-tools` and `u-boot-imx-mfgtools` ought to be upgraded together. So please squash the following two commits into this one.

Reply:
Done